routes/assignment_routes.js

```diff
@@ -0,0 +1,42 @@
+var express = require('express');
+var handleError = require(__dirname + '/../lib/handle_server_error.js');
+var jsonParser = require('body-parser').json;
+var Assignment = require(__dirname + '/../models/assignment.js');
+
+var assignmentRouter = module.exports = exports = express.Router();
+
+assignmentRouter.get('/all-assignments', function(req, res) {
```

Comment:
for now, let's just leave this as a get to `'assignments'`, we can worry about an all assignments when we get the auth in there.